cli/azd/resources/apphost/templates/resources.bicept

```diff
@@ -86,6 +96,16 @@ resource {{mergeBicepName $name $volume.Name}}FileShare 'Microsoft.Storage/stora
   }
 }
 {{- end }}
+{{- range $bMount := $value.BindMounts}}
+resource {{mergeBicepName $name $bMount.Name}}FileShare 'Microsoft.Storage/storageAccounts/fileServices/shares@2022-05-01' = {
+  parent: storageVolumeFileService
+  name: take('${toLower('{{$name}}')}-${toLower('{{removeDot $bMount.Name}}')}', 32)
+  properties: {
+    shareQuota: 1024
```

Comment:
Going to guess setting a quota here is something reasonable for us to do, just calling it out.

Reply:
got it from the docs-sample from aca on mounting file-share :)